src/bin/commands/new-project/utils/utils.ts

```diff
@@ -28,7 +28,7 @@ export function getCommands(
   return {
     installDependencies: () => runTask(packageManager, cwd, "install"),
     lint: () => runTask(packageManager, cwd, "run lint"),
-    test: () => runTask(packageManager, cwd, "test"),
+    test: () => runTask(packageManager, cwd, "test -- -u"),
```

Comment:
This change is curious, I can't see anything in the [Jest v28 release notes](https://jestjs.io/blog/2022/04/25/jest-28) about it... 🤔

Reply:
I *think* it is [this](https://github.com/facebook/jest/pull/12378), which is listed in the 'fixes' section unfortunately (when really it is a breaking change) but it's not entirely clear.